.github/workflows/main.yml

```diff
@@ -38,8 +38,6 @@ jobs:
     environment: sandbox
     steps:
       - uses: actions/checkout@v3
-        with:
-          ref: '3.3.x'
```

Comment:
I removed this in an effort to run actions on my forked repo.  It looks like there's a branching naming convention I might not have adhered to here, so this does bring up a larger question related to the merge: should I rename my branch to 3.3.1?  I may be misunderstanding how actions work, so if you're aware of any other alternative setups that would help me achieve running tests in my fork I'd be happy to hear of any!  

I'll leave this one open for now in case you've got any suggestions - my intention will be to resolve this issue once all the items below have been taken care of (and I'm able to validate my branch is passing PHPUnit tests successfully) - so long-term I don't intend to keep this change.